components/northflank/actions/get-projects/get-projects.mjs

```diff
@@ -0,0 +1,36 @@
+import app from "../../northflank.app.mjs";
+
+export default {
+  key: "northflank-get-projects",
+  name: "Get Projects",
+  description: "Lists all projects with pagination. [See the documentation](https://northflank.com/docs/v1/api/projects/list-projects)",
+  version: "0.0.6",
+  type: "action",
+  props: {
+    app,
+    perPage: {
+      propDefinition: [
+        app,
+        "perPage",
+      ],
+    },
+    page: {
+      propDefinition: [
+        app,
+        "page",
+      ],
+    },
```

Comment:
Props `perPage` and `page` shouldn't be here. You need to use pagination to loop through and list all projects.
```suggestion
```